source/developers-guide/legacy-plugin-system/index.md

```diff
@@ -0,0 +1,820 @@
+### Plugin Configuration / Forms
+
+Backend plugin configuration can be extended by the usage of `$this->Form()->setElement()` in the install method of the plugin bootstrap.php. 
+
+```php
+$form = $this->Form();
+
+// Now we can use this instance of Shopware\Models\Config\Form in $form to add elements to it via `setElement().
+
+$form->setElement(
+    
+    'color', 
+    'yourSetting',
+    [
+        'label' => 'Your label',
+        'description' => Lorem ipsum ...',
+        'value' => '#ffffff',
+        'scope' => Element::SCOPE_SHOP,
+    ]
+);
+```
+
+A textfield would be defined as followed:
+```php
+public function createConfiguration()
+{
+    $form = $this->Form();
+ 
+    $form->setElement('text', 'simpleTextField',
+        array(
+            'label' => 'Text',
+            'value' => 'Preselection',
+            'scope' => Shopware\Models\Config\Element::SCOPE_SHOP,
+            'description' => 'Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut',
+            'required' => true
+        )
+    ); 
+}
+```
+
+#### Options parameter
+The options parameter of the setElement function allows to set several configurations on the form element.
+* **Label**
+The label parameter allows to create a simple descriptional label for the form element.
+* **Value**
+The value parameter stands for the default value of the field if this hasn´t been edited yet. It will directly be shown in the configuration element.
+* **Scope**
+With help of the scope parameter it is possible to generate subshop specific configurations. Leaving this option out results in a configuration option that applies for all subshops.
+* **Description**
+The description parameter allows to provide a more detailed description of the configuration element.
+* **Required**
+The required parameter specifies whether the configuration item is mandatory or not.
+
+#### Possible elements: 
+* color
+* date
+* datetime 
+* html 
+* interval 
+* mediaselection, 
+* number, 
+* select, 
+* text, 
+* textarea 
+* time
+
+#### Read the configuration
+To read out the configuration of your plugin use this code snippet in your plugin bootstrap:
+
+```php
+$setting = $this->Config()->get('yourSetting')
+```
+
+### Backend Menu Items
+
+Example in the install method of the plugin bootstrap.php:
+
+```php
+$this->createMenuItem(
+    [
+        'label' => 'Your plugin laben',
+        'controller' => 'your backend controller',
+        'class' => 'your-icon',
+        'action' => 'Index',
+        'active' => 1,
+        'parent' => $this->Menu()->findOneBy(['controller' => 'Marketing']),
+    ]
+);
+```
+
+For available parent controllers take a look into the table `s_core_menu` (column `controller`). For example you can use one of the following:
+- Article
+- Content
+- Customer
+- ConfigurationMenu
+- Marketing
+
+To know which class for which icon take a look at the <a href="/designers-guide/backend-icons/">Backend icon set overview</a>.
```

Comment:
set {{ site.url }} in front of /designers-guide/backend-icons/

Reply:
d